src/sagas/giphy.js

```diff
@@ -0,0 +1,96 @@
+import { call, takeEvery, put,takeLatest  } from 'redux-saga/effects';
```

Comment:
You might want to separate this sagas into different files to keep it well organized.

Reply:
Thanks for the comments, 
Could you correct all these comments? Or are they just informative?

Reply:
You can correct them if you want, your grades are already calculated though.